src/Microsoft.Health.Fhir.CosmosDb/Configs/CosmosDataStoreConfiguration.cs

```diff
@@ -46,5 +46,7 @@ public class CosmosDataStoreConfiguration
         /// This time includes the time to fetch the first page.
         /// </summary>
         public int SearchEnumerationTimeoutInSeconds { get; set; } = 30;
+
+        public HashSet<string> SortSearchParameters { get; } = new HashSet<string>();
```

Comment:
Suggest adding a comment that these are expected to be the URIs of the search parameters